crates/factors/src/prepare.rs

```diff
@@ -1,6 +1,8 @@
 use std::any::Any;
 
-use crate::{AppComponent, Error, Factor, RuntimeFactors};
+use spin_app::AppComponent;
```

Comment:
Would it make sense to `pub use` this at the `factors` crate level like we're doing for `spin_app::App`?